daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -133,10 +133,31 @@ class TransactionCoderSpec
       }
     }
 
+    "do NodeRollback" in {
```

Comment:
Once we add the version checks we should also add some tests for the unhappy path. choice observers are probably another good example for that type of test.

Reply:
added a test for the unhappy encode path